### ISSUE TITLE
Mobile Display

### ISSUE THREAD

**Issue by jedwardwalters** [srophe/e-gedsh#222]
I just browsed through e-gedsh on my phone to look at mobile display. Most of it looks very good. One thing I wanted to ask about though: on any individual entry page (Such as Aba: http://wwwb.library.vanderbilt.edu/exist/apps/e-gedsh/entry/Aba), the "Browse" menu that shows up on the left hand column in browser display is at the top in the mobile display. So, you have to scroll all the way down through the alphabet browse to get to the entry.

So, in other words, here's what you see on mobile when you go to the Aba entry:
![image-1](https://user-images.githubusercontent.com/11543800/39192781-fb53d7d2-47a7-11e8-8c02-010cea581c1b.jpg)

If it's possible, it would be great if the content of the entry could somehow show up first in mobile display. Or, as an alternative, if that browse menu were collapsable. 

Sorry to drop this in at the last minute, but I at least wanted to raise the issue.

**Comment by dlschwartz** [comment]
Thanks Jamey. Can you take a look @wsalesky? Thanks.

**Comment by wsalesky** [comment]
@jedwardwalters Thanks, this is worth fixing, as it will be a problem across the Syriaca.org apps.

**Comment by wsalesky** [comment]
@jedwardwalters Fixed. Please double check on your phone.

**Comment by jedwardwalters** [comment]
Looks great. The title at the top is getting cut off a little, as you can see here:

**Comment by wsalesky** [comment]
@jedwardwalters Can you repost the image, I don't think it came through.

**Comment by jedwardwalters** [comment]
Oh it must not have worked because I did it as an attachment to an email from my phone. I’ll post the image on the issue directly as soon as I get home.

**Comment by jedwardwalters** [comment]
Here's the image.
![img_3013](https://user-images.githubusercontent.com/11543800/39206971-cdcb423a-47cc-11e8-8371-8bef27111be4.PNG)

**Comment by wsalesky** [comment]
@jedwardwalters Thanks.

**Comment by wsalesky** [comment]
@jedwardwalters this is no longer cut off on my phone. If it looks okay on yours we can close it.

**Comment by jedwardwalters** [comment]
I’m still getting it. And I did a hard refresh. I also checked in both safari and chrome, and it looks the same. 

It’s fine in “landscape” view, but not the regular view.

**Comment by wsalesky** [comment]
@jedwardwalters  That should do it. Close if yes, ping me if no.

**Comment by dlschwartz** [comment]
It looks great to me.

**Comment by dlschwartz** [comment]
@jedwardwalters Are you satisfied with the changes here? If so, go ahead and close the issue. Thanks

**Comment by jedwardwalters** [comment]
Well, let's just assume it's a cache issue on my phone. I'm sure it's fine. I'll close it now.

**Comment by jedwardwalters** [comment]
I just realized something. The issue is not on the index page—that one is fine. It’s the other pages (browse, the entry page for Aba, etc)

**Comment by dlschwartz** [comment]
Yeah, I have the same thing. Landing page is good but everything else has the last line of the title cut in half.

**Comment by wsalesky** [comment]
Okay, fixed interior pages as well.

**Comment by jedwardwalters** [comment]
Looks great!
